guide/web-streaming.md

```diff
@@ -0,0 +1,65 @@
+## Web Streaming
+
+### Description
+
+Webots can be used as a Web streaming server, i.e.,
+to stream a simulation to several interactive 3D `HTML` pages,
+as shown in the [figure bellow](web-streaming.md#screenshot-of-webots-streaming-server).
```

Comment:
typo: bellow -> below

sometimes the 'the' is part of the link and sometimes not. We should be consistent.